view/adminhtml/templates/form/virtual-terminal.phtml

```diff
@@ -19,20 +19,40 @@ $url = $block->getUrl('rvvup/availability/index');
           id="payment_form_<?= $escaper->escapeHtmlAttr($code) ?>"
           style="display:none;">
     <div class="admin__field _required">
-        <label class="label admin__field-label" style="font-weight: normal; display: contents; word-break: auto-phrase;">
+        <label class="label admin__field-label"
+               style="font-weight: normal; display: contents; word-break: auto-phrase;">
             Available after placing order
         </label>
     </div>
-    <script>
-        require(['jquery', 'Rvvup_Payments/js/available-payment'], function($, availablePayment) {
+    <?php if ($secureRenderer): ?>
```

Comment:
Secure renderer is available for >2.4.0, so we can don't need to duplicate the logic.